doc/book/connectors/__java.rst

```diff
@@ -2,4 +2,26 @@
                             Java
 =====================================================================
 
-See http://github.com/tarantool/tarantool-java/.
+There are two Java connectors available:
+
+*   `cartridge-java <http://github.com/tarantool/cartridge-java/>`__
+    supports both single Tarantool nodes and clusters,
+    as well as applications built using the :doc:`Cartridge framework </book/cartridge>` and its modules.
+    Tarantool team actively updates this module with newest Tarantool features.
+*   `tarantool-java <http://github.com/tarantool/tarantool-java/>`__
+    works with early Tarantool versions (1.6 and later),
+    and offers JDBC interface support for single Tarantool nodes.
+    This module has *no active maintenance* and
```

Comment:
```suggestion
    This module *isn't currently maintained* and
```

Reply:
I used "no active maintenance" as a marker here since this combination was used in another place of the text. I think we need some common "deprecated" marker for such things.